src/store/modules/keycodes/kb-settings.js

```diff
@@ -75,6 +75,76 @@ export default [
     title: 'Swap Ctrl and GUI on both sides (for macOS)',
     width: 1500
   },
+  {
+    name: 'Swap Caps/Esc',
+    code: 'EC_SWAP',
+    title: 'Swap Caps Lock and Escape',
+    width: 1500
+  },
+  {
+    name: 'Split Master is Left Hand',
+    code: 'EH_LEFT',
+    title: 'Set the master half of a split keyboard as the left hand (for EE_HANDS)',
+    width: 2000
+  },
+  {
+    name: 'Split Master is Right Hand',
+    code: 'EH_RGHT',
+    title: 'Set the master half of a split keyboard as the right hand (for EE_HANDS)',
+    width: 2000
+  },
+  {
+    width: 0
+  },
+  {
+    name: 'Toggle LCtl/Caps',
+    code: 'CL_TOGG',
+    title: 'Toggle Caps Lock and Left Control swap',
+    width: 1500
+  },
+  {
+    width: 3000
+  },
+  {
+    width: 3000
+  },
+  {
+    width: 1500
+  },
```

Comment:
Not sure how I feel about this bit...

Reply:
You could create an alias to an object like so:
```
const blank3000Row = { width: 3000 };
```
and then use it like so:
```javascript
...bank3000Row
```

Reply:
The issue I'm having with this is that I need a space that's 7.5 key units, but there isn't a spacer class in the CSS for one.

I could add one of course, but I don't think that's a scalable solution.